lib/dash/dash_parser.js

```diff
@@ -383,6 +383,13 @@ shaka.dash.DashParser = class {
     }
 
     // The segments are available earlier than the availability start time.
+    if (this.minTotalAvailabilityTimeOffset_ && !this.lowLatencyMode_) {
+      const autoLowLatencyMode = this.playerInterface_.isAutoLowLatencyMode();
+      if (autoLowLatencyMode) {
```

Comment:
Please add comments here to explain how lowLatencyMode & autoLowLatencyMode works.

Reply:
Done!